dot/rpc/modules/dev_test.go

```diff
@@ -122,19 +135,28 @@ func TestDevModule_SlotDuration(t *testing.T) {
 			},
 			args: args{
 				req: &EmptyRequest{},
-				res: &res,
 			},
+			exp: "0x1700000000000000",
 		},
 	}
 	for _, tt := range tests {
+		var res string
+		tt.args.res = &res
```

Comment:
move this inside the subtest. Once you run (in the future) parallel subtests, each run in their own goroutine so it's rather hard to ensure they don't access some shared pointer etc.

TLDR, don't put anything between the `for` and the `func(t *testing.T) {` of the subtest :wink: